streams/src/test/java/org/apache/kafka/streams/processor/internals/AbstractTaskTest.java

```diff
@@ -72,27 +64,6 @@ public void before() {
         expect(stateDirectory.directoryForTask(id)).andReturn(TestUtils.tempDirectory());
     }
 
-    @Test(expected = ProcessorStateException.class)
-    public void shouldThrowProcessorStateExceptionOnInitializeOffsetsWhenAuthorizationException() {
-        final Consumer consumer = mockConsumer(new AuthorizationException("blah"));
-        final AbstractTask task = createTask(consumer, Collections.<StateStore, String>emptyMap());
-        task.updateOffsetLimits();
-    }
-
-    @Test(expected = ProcessorStateException.class)
-    public void shouldThrowProcessorStateExceptionOnInitializeOffsetsWhenKafkaException() {
-        final Consumer consumer = mockConsumer(new KafkaException("blah"));
-        final AbstractTask task = createTask(consumer, Collections.<StateStore, String>emptyMap());
-        task.updateOffsetLimits();
-    }
-
-    @Test(expected = WakeupException.class)
-    public void shouldThrowWakeupExceptionOnInitializeOffsetsWhenWakeupException() {
-        final Consumer consumer = mockConsumer(new WakeupException());
-        final AbstractTask task = createTask(consumer, Collections.<StateStore, String>emptyMap());
-        task.updateOffsetLimits();
-    }
```

Comment:
Seems we should add corresponding test for new method `committedOffsetForPartition(final TopicPartition partition)` ?

Reply:
I end up testing this with the migrated StreamTaskTests and with the soon to be added StandbyTaskTests.